Client/Client.csproj

```diff
@@ -36,17 +43,26 @@
     <ProjectReference Include="..\Core\Core.csproj" />
   </ItemGroup>
 
+  <Target Name="SetPublishDir" BeforeTargets="Build">
+    <PropertyGroup>
+      <PublishDir Condition="'$(SelfContainedRelease)' == 'true'">$(HelionRootDir)\Publish\$(RuntimeIdentifier)_SelfContained\</PublishDir>
```

Comment:
I'm redirecting the publish directory closer to the repo root to keep the publish paths shorter and easier to find.